diff --git a/Dumper/Generator/Private/Managers/StructManager.cpp b/Dumper/Generator/Private/Managers/StructManager.cpp
index c7ea1f5..16e0ca5 100644
--- a/Dumper/Generator/Private/Managers/StructManager.cpp
+++ b/Dumper/Generator/Private/Managers/StructManager.cpp
@@ -249,8 +249,12 @@ void StructManager::Init()
 	InitAlignmentsAndNames();
 	InitSizesAndIsFinal();
 
+	/* Fix FDataTableRowHandle */
+	auto rowHandleStructInfo = &StructInfoOverrides[ObjectArray::FindStructFast("DataTableRowHandle").GetIndex()];
+	rowHandleStructInfo->Size = 0x10;
+	rowHandleStructInfo->Alignment = 0x8;
+
 	/*
 	* The default class-alignment of 0x8 is only set for classes with a valid Super-class, because they inherit from UObject.
 	* UObject however doesn't have a super, so this needs to be set manually.
