diff --git a/SDK-GRTS/SDK/SDK/OEIOnlineSubsystemUtils_structs.hpp b/SDK-GRTS/SDK/SDK/OEIOnlineSubsystemUtils_structs.hpp
index cc6290c..073926f 100644
--- a/SDK-GRTS/SDK/SDK/OEIOnlineSubsystemUtils_structs.hpp
+++ b/SDK-GRTS/SDK/SDK/OEIOnlineSubsystemUtils_structs.hpp
@@ -82,7 +81,7 @@ enum class EOnlinePermissionDenyReason : uint8
 
 // ScriptStruct OEIOnlineSubsystemUtils.DataTableRowHandle_NetCrc32
 // 0x0020 (0x0030 - 0x0010)
-struct FDataTableRowHandle_NetCrc32 final : public FDataTableRowHandle
+struct FDataTableRowHandle_NetCrc32 final
 {
 public:
 	struct FDataTableRowHandle                    DataTableRowHandle;                                // 0x0000(0x0010)(BlueprintVisible, BlueprintReadOnly, Transient, NoDestructor, NativeAccessSpecifierPublic)
@@ -94,7 +93,7 @@ static_assert(offsetof(FDataTableRowHandle_NetCrc32, DataTableRowHandle) == 0x00
 
 // ScriptStruct OEIOnlineSubsystemUtils.DataTableRowHandle_NetCrc64
 // 0x0028 (0x0038 - 0x0010)
-struct FDataTableRowHandle_NetCrc64 final : public FDataTableRowHandle
+struct FDataTableRowHandle_NetCrc64 final
 {
 public:
 	struct FDataTableRowHandle                    DataTableRowHandle;                                // 0x0000(0x0010)(BlueprintVisible, BlueprintReadOnly, Transient, NoDestructor, NativeAccessSpecifierPublic)
